### ISSUE TITLE
Suscripción delivery propio

### ISSUE THREAD

**Issue by masoto22** [IIC2413/Syllabus-2024-1#150]
Hola

Espero que estén bien. Escribo porque modelando las nuevas reglas del negocio, nos surgió la duda si los clientes también pueden suscribirse a los delivery propios de los restaurantes? (Esto considerando que anteriormente los clientes podían suscribirse a la empresa despachadora (ahora llamada Delivery).

Gracias!

**Comment by ypne** [comment]
No, solo los delivery tienen suscripción. Los restaurantes despachan gratis sobre el monto mínimo de pedido
